app/apps/server/storage/AppFileSystemSourceStorage.ts

```diff
@@ -2,26 +2,28 @@ import { promises as fs } from 'fs';
 
 import { AppSourceStorage, IAppStorageItem } from '@rocket.chat/apps-engine/server/storage';
 
-const tempPath = '/Users/douglasgubert/dev/rocket.chat/Rocket.Chat/apps-engine-files/';
-
 export class AppFileSystemSourceStorage extends AppSourceStorage {
+	constructor(private storagePath: string) {
```

Comment:
We still need a method in this class to change this `storagePath` property according to the value of the setting

Reply:
Yes! It is done